Xamarin.Forms.Core/Label.cs

```diff
@@ -32,6 +32,8 @@ public class Label : View, IFontElement, ITextElement, ITextAlignmentElement, IE
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
 
+		public static readonly BindableProperty TextDecorationsProperty = BindableProperty.Create("TextDecorations", typeof(TextDecorations), typeof(Label), TextDecorations.None);
```

Comment:
You can actually use nameof(TextDecorations) now. They just didn't use it before, because XF started before the feature was added to C#

Reply:
sure, I'm aware of the feature but was trying to copy existing code styles. Thanks!

Reply:
What I was trying to say, is the XF team are happy for you to start using the nameof feature now :) They are hoping to update all in the future.